tests/check_pythonload.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+import unittest
+import imp
+import glob
+import os, sys
+
+# Skip individual modules based on module name.  For example,
+# pyupm_ozw will skip pyupm_ozw.py.
+blacklist = [
+             # OpenZwave-dependent modules
+             'pyupm_ozw', 'aeotecdw2e', 'tzemt400', 'aeotecdsb09104',
+             'aeotecss6', 'aeotecsdg2', 'ozwdump',
+             # Requires mraa python module
+             'curieimu',
+             # Requires modbus
+             'h803x', 'hwxpxx', 't3311',
+             # Requires bacnet
+             'e50hx', 'tb7300', 't8100',
+             # Requires PIL
+             'make_oled_pic'
+             ]
+
```

Comment:
Would it be better to have these in a datafile rather than hardcoded into a script?